Source/config/DBsampledata.js

```diff
@@ -1,223 +0,0 @@
-var Building = require('../app/models/building-schema');
-
```

Comment:
Is there a reason we got rid of this file? Did you create a new way to add all of the buildings?

Reply:
I got rid of this file because we can now add our own buildings. Although now that I think about it, the building schemas probably aren't the same from your template.

Reply:
Change of schemas is fine. Was just concerned with manually adding all the buildings again, but I dont think its avoidable